server/controllers/auth.js

```diff
@@ -111,3 +111,8 @@ exports.registerController = async (req, res, next) => {
     }
   }
 };
+
+exports.logout = (req, res) => {
+  res.clearCookie('token');
+  res.status(200).json({ status: 200, message: 'logout success' });
```

Comment:
by default status code will be `200` so you don't need to use` res.status()`
```suggestion
  res.json({ status: 200, message: 'logout success' });
```